pkgs/top-level/perl-packages.nix

```diff
@@ -13045,7 +13045,10 @@ let
       description = "The World-Wide Web library for Perl";
       license = with licenses; [ artistic1 gpl1Plus ];
     };
-    buildInputs = [ TestFatal TestNeeds TestRequiresInternet ];
+    buildInputs = [ TestFatal TestNeeds TestRequiresInternet ]
+      # cross-compilation: avoid test dependency HTTPDaemon if we're cross
+      # compiling. perl-cross miniperl isn't able to build it.
+      ++ lib.optionals (stdenv.buildPlatform == stdenv.hostPlatform) [ HTTPDaemon ];
```

Comment:
Why not just use `checkInputs`?

Reply:
That's a good question: The convention in perl-packages.nix has always been to include test and build dependencies in `buildInputs`. I'll update this PR.

PS: In the case of `LWP` it's listed under `build_requires` in [META.yml](https://metacpan.org/release/OALDERS/libwww-perl-6.67/source/META.yml) and under `prereqs.test.requires` in [META.json](https://metacpan.org/release/OALDERS/libwww-perl-6.67/source/META.json).